subprojects/docs/src/snippets/testing/testReport/groovy/build.gradle

```diff
@@ -22,8 +22,8 @@ dependencies {
 }
 
 tasks.register('testReport', TestReport) {
-    destinationDir = layout.buildDirectory.dir('reports/allTests').get().asFile
+    destinationDirectory = layout.buildDirectory.dir('reports/allTests').get().asFile
```

Comment:
This shouldn't be using get().asFile since this removes the provider aspect of it